### PR TITLE
Add QRCode feature for bottom toolbar

### PR DESCRIPTION
**Pull request by codeimpl**
This patch add QRCode feature to bottom toolbar. But this feature
location is just temporary, it can be changed soon.

Refs: #886

### CODE REVIEW

File: banana_cake/java/org/banana/cake/CakeToolbarManager.java [hunk 1]
```diff
@@ -179,4 +180,13 @@ public void closeCurrentTab() {
         if (activity == null) return;
         activity.onMenuOrKeyboardAction(R.id.close_tab, false);
     }
+
+    @Override
+    public void openQRCodeDialog() {
+        ChromeTabbedActivity activity = getChromeTabbedActivity();
+        if (activity == null) return;
+        QrCodeCoordinator qrCodeCoordinator =
+                new QrCodeCoordinator(activity, activity.getActivityTab().getUrl().getSpec());
```
Comment (romandev) [inline banana_cake/java/org/banana/cake/CakeToolbarManager.java:189]: The getActivityTab() is nullable. Therefore, we should check whether it is null or not.

Comment (romandev) [reply banana_cake/java/org/banana/cake/CakeToolbarManager.java:189]: If the url is empty, what happens?

Comment (codeimpl) [reply banana_cake/java/org/banana/cake/CakeToolbarManager.java]: > The getActivityTab() is nullable. Therefore, we should check whether it is null or not.

done!

Comment (codeimpl) [reply banana_cake/java/org/banana/cake/CakeToolbarManager.java]: > If the url is empty, what happens?

qrcode will be null, it means that user can't get qrcode.

**Review by romandev — APPROVED** [review]
lgtm

**Pull request merged**